package.json

```diff
@@ -57,6 +57,7 @@
     "next-mdx-enhanced": "^2.4.0",
     "prettier": "^1.19.1",
     "react": "^16.12.0",
+    "react-collapsible": "^2.7.0",
```

Comment:
Good choice of a library! Seems like it was pretty easy to implement and looks like it's working really well 💯